web/src/pages/predictions.js

```diff
@@ -25,6 +25,9 @@ export const query = graphql`
 
 const PredictionsPage = (props) => {
   const { data, errors } = props;
+  const { auth } = firebase;
+  const [user, setUser] = useState();
+  useEffect(() => auth().onAuthStateChanged(setUser), []);
```

Comment:
Maybe it would make sense to use providers and context to pass user data throughout all the predictions game components